pkg/bootstrap/stage.go

```diff
@@ -165,3 +174,30 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 	}
 	return os.Rename(tempDir, dir)
 }
+
+func preloadBootstrapImage(dataDir, runtimeImage string) (v1.Image, error) {
+	imagesDir := filepath.Join(dataDir, "agent", "images")
+	files := map[string]os.FileInfo{}
+	if err := filepath.Walk(imagesDir, func(path string, info os.FileInfo, err error) error {
```

Comment:
This is a little hard to scan - is it worth assigning the anonymous function to a local variable and then passing that into Walk to improve readability?

Reply:
I think thats the common pattern for calling filepath.Walk() and I found several instances in the code with the same pattern, but I can see your point of making the code more readable